### DISCUSSION TITLE
After using make.pheatmap function, ggplots no longer generates previews

### DISCUSSION THREAD

**Discussion by josephangelolopez** [ImmuneDynamics/Spectre#137] [Help]
Hi Tom,

I'm finding that when I use the make.pheatmap function to generate my expression heatmaps, whenever I try to make new plots of anything via ggplot, they no longer show up on the Plots window in RStudio and I'm having to restart my R session and reload packages in order to get my plot previews working again.

Was wondering if this is a known issue and if there's a workaround for this? Also note, unlike the other functions like make.colour.plot and make.multi.plot, the heatmaps do not generate a preview in RStudio.

Thanks!

**Comment by hrj21** [comment]
I can confirm I see the same behaviour. It's almost as if the graphics device isn't closed properly. A quick work around for now is to run `graphics.off()` and then any code after this should produce a preview in rstudio.

**Reply by tomashhurst** [reply]
@josephangelolopez @hrj21 Well spotted! Hmm that is odd. I'll have a more detailed look at the code, but good find @hrj21 with the `graphics.off()` option.
